README.md

```diff
@@ -556,7 +560,10 @@ class Machine
       define_method(action) { object.send(action) } if action
     end
 
-    machine_class.new
+    # This won't change the value of the `attribute` on the Vehicle class if it already
+    # has a value assigned to it. Otherwise it will set it to the initial one
+    state = object.send(attribute)
+    machine_class.new.tap{|machine| machine.send("#{attribute}=", state) if state}
```

Comment:
Do we need to complicate the example that much? Why the previous example is not good?

Reply:
Ok I really don't know how I've missed this notification...I'm gonna reply back as soon as I get off from work. Sorry.